packages/contracts-bedrock/test/dispute/FaultDisputeGame.t.sol

```diff
@@ -449,6 +449,13 @@ contract FaultDisputeGame_Test is FaultDisputeGame_Init {
         gameProxy.initialize();
     }
 
+    /// @dev Tests that startingOutputRoot and it's getters are set correctly.
+    function test_startingOutputRootGetters_succeeds() public view {
```

Comment:
Let's also check that the values here match the values that were provided in the constructor in the setup function